evap/contributor/templates/contributor_evaluation_form.html

```diff
@@ -26,7 +26,7 @@ <h3>
         {{ evaluation.full_name }} ({{ evaluation.course.semester.name }})
```

Comment:
- Comma as auto-separator for tags should be removed (so that a comma can be part of text answer warnings)